scopes/harmony/aspect-loader/plugins.ts

```diff
@@ -21,6 +23,7 @@ export class Plugins {
   //     return plugin.def.dependencies;
   //   });
   // }
+  private static checkedComponents: Set<string> = new Set();
```

Comment:
This is not really checked components but non plugin components.
You only add components that are not plugins. Not any component you visited.